R/simulate.r

```diff
@@ -1,4 +1,4 @@
-#' Simulate a tree of infections with a serial and offspring distributions
+#' Simulate transmission trees from an initial number of infections
 #'
 #' @param nchains Number of chains to simulate.
```

Comment:
```suggestion
#' @param ntrees Number of trees to simulate.
```

Reply:
This will break a lot of things including the tests and vignettes, even though it'll require the exact change throughout. I think it's better done in a separate PR.